From f3c3f2b380431f6b46008838d82e056ea6e05d6b Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Tue, 31 Oct 2023 16:34:56 +0000
Subject: [PATCH] fixup! arch/arm64: Add Revision, Serial, Model to cpuinfo

Delete the Hardware string, which is pointless and misleading.

See: https://github.com/raspberrypi/bookworm-feedback/issues/129

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/kernel/cpuinfo.c | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm64/kernel/cpuinfo.c
+++ b/arch/arm64/kernel/cpuinfo.c
@@ -224,8 +224,6 @@ static int c_show(struct seq_file *m, vo
 		seq_printf(m, "CPU revision\t: %d\n\n", MIDR_REVISION(midr));
 	}
 
-	seq_printf(m, "Hardware\t: BCM2835\n");
-
 	np = of_find_node_by_path("/system");
 	if (np) {
 		if (!of_property_read_u32(np, "linux,revision", &revision))
